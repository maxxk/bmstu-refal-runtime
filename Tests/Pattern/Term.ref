$ENTRY Go
{
	= <Prout <Func1 a b c d>>
	  <Prout <Func1 (a) b (c) d>>
	  <Prout <Func1 ((a) b) (d f) sd (c) d>>
	  <Prout <Func1 (a) b (d f) sd (c) d>>;
}

Func1
{
	t.1 t.2 t.3 e.1 = t.1 t.2 t.3;
}

