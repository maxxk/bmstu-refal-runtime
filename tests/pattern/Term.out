a b c 
(a )b (c )
((a )b )(d f )sd 
(a )b (d f )
